main.go

```diff
@@ -107,10 +138,10 @@ func main() {
 		setupLog.Error(err, "unable to set up ready check")
 		os.Exit(1)
 	}
+}
 
-	setupLog.Info("starting manager")
-	if err := mgr.Start(ctrl.SetupSignalHandler()); err != nil {
-		setupLog.Error(err, "problem running manager")
-		os.Exit(1)
+func setupIndexes(mgr ctrl.Manager) {
```

Comment:
no need for this function, it is only a one liner.

Reply:
done